draft-ietf-taps-interface.md

```diff
@@ -3054,8 +3073,11 @@ coverage, see {{prop-checksum-control-send}} and {{prop-checksum-control-receive
 * "Specify minimum checksum coverage required by receiver" and "Disable checksum requirement when receiving":
 `Required Minimum Corruption Protection Coverage for Receiving` property ({{conn-recv-checksum}}) and `Full Checksum Coverage on Receiving` property ({{prop-checksum-control-receive}}).
 
-* "Specify DF" field and "Request not to bundle messages":
-the `No Fragmentation` Message Property combines both of these requests, i.e. if a request not to bundle messages is made, this also turns off fragmentation (i.e., sets DF=1) in the case of a protocol that allows this (only UDP and UDP-Lite, which cannot bundle messages anyway) ({{send-singular}}).
+* "Specify DF field":
+`No Network-Layer Fragmentation` property ({{send-singular}}).
```

Comment:
Still slightly unhappy with the "v4-centric" words of "DF field" and would prefer something different and definitely to omit "field"

Reply:
Oh, this shouldn't change, it's a reference to minset, where this is the exact wording!